device/dell/x86_64-dellemc_z9332f_d1508-r0/plugins/sfputil.py

```diff
@@ -206,6 +202,8 @@ def set_low_power_mode(self, port_num, lpmode):
         # Check for invalid port_num
         if port_num < self.port_start or port_num > self.port_end:
             return False
+        if port_num > self.PORTS_IN_BLOCK:
```

Comment:
power override bit 0 in byte 93 must be '0' (see SFF 8636) before LPMode signal can take control of low power mode